content/en/docs/concepts/services-networking/_index.md

```diff
@@ -7,44 +7,195 @@ description: >
-"IP-per-pod" model.
-
-How this is implemented is a detail of the particular container runtime in use.
-
-It is possible to request ports on the `Node` itself which forward to your `Pod`
-(called host ports), but this is a very niche operation. How that forwarding is
-implemented is also a detail of the container runtime. The `Pod` itself is
-blind to the existence or non-existence of host ports.
+* Containers in same Pod can communicate with each other.
+* Pods can communicate with all other Pods on the same or separate [nodes](/docs/concepts/architecture/nodes/)
+without network address translation (NAT).
+* Pods can communicate with other Pods on the same or separate nodes using layer 2 bridging, native layer 3 (IPv4, IPv6) networking, MPLS, segment routing, tunneling, or other techniques for conveying packets from one Pod to another.
+
+Some platforms, such as Linux and Windows, support Pods running in the host network. Pods attached to the host network of a node can still communicate with all Pods on all nodes without NAT.
+
+{{< note >}}
+Some of the terms used in describing K8s networking might be new or require more details. To review these terms, see [K8s network terminology](#terminology).
+{{< /note >}}
+
+### Containers on the same pod
+
+Inter-container connectivity on the same Pod makes use of pod IPs, network namespaces and localhost networking.
+
+Pods run in their own [network namespace](https://man7.org/linux/man-pages/man7/network_namespaces.7.html). All the containers in a Pod share this network namespace. (Network namespaces are not the same as the Kubernetes {{< glossary_tooltip text="namespace" term_id="namespace" >}} concept).
+
+Figure 2 illustrates an example of localhost communications between two containers on the same Pod. 
+
+{{< figure src="/docs/images/k8s-localhost-02.drawio.svg" alt="diagram of K8s local networking for containers in a single Pod" class="diagram-large" caption="Figure 2. Example of container communications within the same Pod using localhost:port number" >}}
+
+Your Pod 1 containers share a network namespace, IP address and MAC address. However, you do need to configure separate port numbers for each container.
+
+### Pods on the same node
+
+Inter-pod connectivity on the same node utilize pod IPs, Pod and root network namespaces, [veth links](https://man7.org/linux/man-pages/man4/veth.4.html) and a [virtual L2 bridge](https://en.wikipedia.org/wiki/Data_link_layer).
+
+Pods operate in their own network namespace and have their own IP and MAC addresses. You can think of this as multiple distinct IP computers (pods), that wish to talk with each other through a L2 bridge. Inter-pod networking on the same node is no different.
+
+This example of inter-pod networking on the same node requires:
+
+* Root network namespace for the node. 
+
+* Veth links enabling communication between the Pod network namespace and root network namespace. 
```

Comment:
Whilst this is true, I'm worried that readers will infer that _all_ inter-pod packet networking requires a veth.

Reply:
There are a large number of K8s network blogs already out there. Would we add any value? New reader sees the components, wonders how this could work with simple examples and then off they go to google or K8s blog? 

I have always appreciated "one stop shop" for architecture and then implementation examples. Now I have "jumping off point" to dig deeper. Networking is hard; examples in close proximity is a must for learning, IMO.  

We have [K8s Components](https://kubernetes.io/docs/concepts/overview/components/). Perhaps I could zoom in to the nodes and add the core network components?

Reply:
I like this diagram; it's just that I'm confident lots of people will misunderstand what we're saying, unless we reframe the explanation.